src/main/java/com/rapid7/container/analyzer/docker/service/DockerImageAnalyzerService.java

```diff
@@ -175,12 +175,24 @@ public <M extends Manifest> Image analyze(File outputDirectory, ImageId id, Dige
     try {
       // parse the image configuration
       List<HistoryJson> layerHistories = configuration.getHistory();
+      if (layerHistories == null) {
+        LOGGER.error("Layer histories either do not exist or there was an error parsing them. Image with id " + id.getId() + " has been SKIPPED.");
```

Comment:
Can use params instead of concatenation for log messages:

```suggestion
        LOGGER.error("Layer histories either do not exist or there was an error parsing them. Image with id {} has been SKIPPED.", id.getId());
```